kolibri/plugins/management/assets/src/vue/manage-class-page/class-create-modal.vue

```diff
@@ -85,6 +86,9 @@
       },
     },
     vuex: {
+      getters: {
+        facility: state => state.facility,
+      },
```

Comment:
I'd also consider having `createClass` explicitly specify parameters (e.g. `name`), rather than bundling them into an ambiguous `stateClassData` object.